packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -70,6 +70,12 @@ function get_base_diffusers_model(model: ModelData): string {
 	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
 }
 
+function get_prompt_from_diffusers_model(model: ModelData): string {
+	return model.cardData?.widget?.[0]?.text?.toString()
+	    ?? model.cardData?.instance_prompt?.toString()
```

Comment:
I don't know where to get this from.

Reply:
this one's fine imo

Reply:
I don't understand how it works though. Is it a part of `inference.parameters`? It's not part of https://huggingface.co/docs/api-inference/tasks/text-to-image

Reply:
I've added `instance_prompt` to `cardData` to fix the typing error, but I'm not completely sure this will make it pull correctly from the model card https://github.com/huggingface/huggingface.js/pull/909/files#diff-4dbb69668b0377cf24081f874e42ce011491bed90939745260b6f65d689dd941R110

Reply:
wdyt @mishig25 ? 

Reply:
> I've added instance_prompt to cardData
checked in moon. What you did is correct 👍

Reply:
instance_prompt is validated to be a string in moon. But no big deal to add a toString